lib/pyld/jsonld.py

```diff
@@ -4697,9 +5242,21 @@ def _find_context_urls(self, input_, urls, replace, base):
                     elif v not in urls:
                         urls[v] = False
                 elif _is_object(v):
-                    # look for soped context
+                    # look for sooped context
```

Comment:
sooped context... nomnom.
```suggestion
                    # look for scoped context
```